sycl/test-e2e/Basic/large-range.cpp

```diff
@@ -0,0 +1,82 @@
+// RUN: %{build} -fno-sycl-id-queries-fit-in-int -D__SYCL_ENABLE_LARGE_RANGE_WRAPPING__ -o %t.out
```

Comment:
Is `__SYCL_ENABLE_LARGE_RANGE_WRAPPING__` something that should be explicitly defined by the user to make this mechanism work?

Reply:
Feature has now been unguarded from the macro and merged into the range rounding feature.